Resources/Textures/Clothing/Head/Hardsuits/piratecaptainhelm.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "created by /Discord: brainfood#7460/github: brainfood1183/ for space-station-14.",
```

same for the 4 or so others that dont follow this
```suggestion
    "copyright": "Made by brainfood1183 (Github) for ss14",
```